R/utils.R

```diff
@@ -438,11 +438,23 @@ listing_format_chevron <- function() {
 #'
 #' @return a `function` converting a date into `string`.
 #'
-#' @export
+#' @note The date is extracted at the location of the measure, not at the location of the system.
 #'
+#' @export
+#' @examples
+#' format_date("%d%b%Y")(as.Date("2021-01-01"))
+#' format_date("%d%b%Y")(as.POSIXct("2021-01-01 00:00:01", tz = "NZ"))
+#' format_date("%d%b%Y")(as.POSIXct("2021-01-01 00:00:01", tz = "US/Pacific"))
```

Comment:
while this is a nice check, but it reminds me if we should use the POSIXct in the listing, or use the variable that is date only?

Reply:
I am not sure, the current data set is in POSIXct so I assumed it was the correct format. I works with both, but for the POSIXct we have to be a bit careful and make sure that the time zone is set.

Reply:
and how about using the TRTSDT instead of TRTSDTM here as the default? (of course this update is a nice one can be kept)

Reply:
I created an issue here: https://github.com/insightsengineering/chevron/issues/752